composer.json

```
@@ -10,9 +10,9 @@
   }
   ],
   "require": {
-    "php": "^7.3||^8.0",
-    "drupal/core": "^8.5||^9.1",
-    "symfony/lock": "^3.4||^4.0"
+    "php": "^8.0",
+    "drupal/core": "^9.1",
```

Maybe we should take the chance to upgrade to D10 compatibility? This will likely have implications on the symfony component versions.

```suggestion
    "drupal/core": "^9.1||^10",
```